bindings/python/project.toml

```
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel"]
```

Shouldn't this be `pyproject.toml` and not `project.toml`? Or are you doing something different that I don't know about?

```suggestion
[build-system]
requires = ["setuptools>=61.0"]
build-backend = "setuptools.build_meta"
```

If you are using a `pyproject.toml` based system in keeping with [PEP 518](https://peps.python.org/pep-0518/) then `build-backend` should be included. In https://scikit-hep.org/developer/pep621#pyprojecttoml-build-system we recommend a modern 

```toml
[build-system]
requires = ["setuptools>=61.0"]
build-backend = "setuptools.build_meta"
```

but maybe there are limitations on this version of `setuptools` for XRootD?

---

>=42 is fine if you need Python 2 support.

---

Yeah, pyproject.toml, that's why I'm asking for your review :-) It works as is, but the misnaming was an oversight. I will fix and push a new version.

---

For 5.x we still need to keep support for Python 2, but when 6.0 comes out, we will drop support for Python 2.x.

---

Fixed in latest version.